antora-ui-camel/src/css/frontpage.css

```diff
@@ -353,13 +366,13 @@ section.frontpage.projects .project img {
     padding: 0;
   }
 
-  section.frontpage.projects .project h2,
+  section.frontpage.projects .project .content h2,
   section.frontpage.projects .project .icon,
   section.frontpage.columns.functionalities .box .content h2,
   section.frontpage.columns.functionalities .box .content,
   section.frontpage.columns.functionalities .box .icon,
   section.frontpage.columns.functionalities p,
-  section.frontpage.projects p {
+  section.frontpage.projects .content p {
```

Comment:
@AemieJ instead of adding a media-code snippet, this line should to be scoped to `section.frontpage.projects .project .content p`. It's getting overwritten by the rule on line 197. Sorry for missing it out before.